polaris-react/src/components/Navigation/Navigation.scss

```diff
@@ -197,26 +198,11 @@ $disabled-fade: 0.6;
   @include nav-item-icon-attributes;
 }
 
-.ExternalIcon {
```

Comment:
`external` was removed in [this PR](https://github.com/Shopify/polaris/pull/6518) so this is no longer needed.